app/src/main/java/com/swent/assos/model/navigation/NavigationGraph.kt

```diff
@@ -1,47 +1,60 @@
 package com.swent.assos.model.navigation
 
 import androidx.compose.runtime.Composable
+import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
-import androidx.navigation.compose.navigation
 import androidx.navigation.compose.rememberNavController
-import com.swent.assos.model.data.Association
+import com.swent.assos.model.view.AppViewModel
 import com.swent.assos.ui.login.LoginScreen
 import com.swent.assos.ui.login.SignUpScreen
 import com.swent.assos.ui.screens.Settings
-import com.swent.assos.ui.screens.manageAssos.AssoDigest
-import com.swent.assos.ui.screens.manageAssos.CreateEvent
-import com.swent.assos.ui.screens.manageAssos.CreateNews
+import com.swent.assos.ui.screens.assoDetails.AssoDetails
+import com.swent.assos.ui.screens.assoDetails.EventDetails
+import com.swent.assos.ui.screens.assoDetails.NewsDetails
+import com.swent.assos.ui.screens.manageAsso.CreateEvent
+import com.swent.assos.ui.screens.manageAsso.CreateNews
+import com.swent.assos.ui.screens.manageAsso.ManageAssociation
 
 @Composable
 fun NavigationGraph() {
   val navController = rememberNavController()
   val navigationActions = NavigationActions(navController = navController)
+  val appViewModel: AppViewModel = hiltViewModel()
 
-  NavHost(navController = navController, startDestination = Destinations.LOGIN.route) {
+  NavHost(navController = navController, startDestination = Destinations.HOME.route) {
```

Comment:
We should login be able to login. I suspect this is debugging code 
```suggestion
  NavHost(navController = navController, startDestination = Destinations.LOGIN.route) {
```